### PR TITLE
Bug 1303375 - Add Accessibility support for the TopSite page control in AS.

### PR DESCRIPTION
**Pull request by farhanpatel**

### CODE REVIEW

File: Client/Frontend/Home/ActivityStreamTopSitesCell.swift [hunk 1]
```diff
@@ -183,7 +184,11 @@ class ASHorizontalScrollCell: UITableViewCell {
         let pageControl = FilledPageControl()
         pageControl.tintColor = UIColor.grayColor()
         pageControl.indicatorRadius = ASHorizontalScrollCellUX.PageControlRadius
-        pageControl.userInteractionEnabled = false
+        pageControl.userInteractionEnabled = true
+        pageControl.isAccessibilityElement = true
+        pageControl.accessibilityIdentifier = "pageControl"
+        pageControl.accessibilityLabel = NSLocalizedString("TopSites Page control", comment: "The page control button that lets you switch between pages.")
```
Comment (sleroux) [inline Client/Frontend/Home/ActivityStreamTopSitesCell.swift]: I would put this in the Top Sites section of `Strings.swift` [1] and follow the same format as the other strings defined there.
[1] https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Strings.swift#L34

File: Client/Frontend/Home/ActivityStreamTopSitesCell.swift [hunk 2]
```diff
@@ -242,6 +254,20 @@ class ASHorizontalScrollCell: UITableViewCell {
         pageControl.progress = currentPage
     }
 
+    func handlePageTap(gesture: UITapGestureRecognizer) {
+        if pageControl.pageCount <= 1 {
```
Comment (sleroux) [inline Client/Frontend/Home/ActivityStreamTopSitesCell.swift]: Nit: Would read better as a `guard` since we're using the if as a guard to the code below.

**Comment by sleroux** [comment]
Oops! Still learning this Github review feature and accidentally hit approve. Probably want to change the string before we merge.

File: Client/Frontend/Strings.swift [hunk 3]
```diff
@@ -36,6 +36,13 @@ extension Strings {
     public static let TopSitesRemoveButtonAccessibilityLabel = NSLocalizedString("TopSites.RemovePage.Button", value: "Remove page - %@", comment: "Button shown in editing mode to remove this site from the top sites panel.")
 }
 
+// Activity Stream.
+extension Strings {
+    public static let ASPageControlButton = NSLocalizedString("AS.PageControl.Button", comment: "The page control button that lets you switch between pages in top sites")
```
Comment (flodolo) [inline Client/Frontend/Strings.swift]: This doesn't have a value, and in the XLIFF is exposed as 'AS.PageControl.Button', which doesn't make a lot of sense

**Pull request merged**